### DISCUSSION TITLE
canale youtube

### DISCUSSION THREAD

**Discussion by tommasiangelo01** [iisgiua/giuaschool#608] [Richieste di aiuto]
Scusate,ma non c'era qualcuno che si era incaricato di aprire un canale youtube e tenerlo operativo con i contenuti inerenti al registro giuaschool?

**Comment by sempliciniluigi89** [comment]
Sull'installazione, aggiornamenti,nuove implementazioni

Il Mer 13 Dic 2023, 23:58 gmontalbano5ct ***@***.***> ha
scritto:
